.github/CODEOWNERS

```diff
@@ -80,9 +80,10 @@ Tools/patchcheck/             @AA-Turner
 # ----------------------------------------------------------------------------
 
 # Autotools
-configure*                       @erlend-aasland @corona10 @AA-Turner
-Makefile.pre.in                  @erlend-aasland @AA-Turner
-Modules/Setup*                   @erlend-aasland @AA-Turner
+configure*                       @erlend-aasland @corona10 @AA-Turner @emmatyping
+Makefile.pre.in                  @erlend-aasland @AA-Turner @emmatyping
+Modules/Setup*                   @erlend-aasland @AA-Turner @emmatyping
+Modules/makesetup                @emmatyping
```

Comment:
Ah we actually wanted to retain alphabetical order here! and I believe we wanted it to be case insensitive (otherwise configure would come at the end since `c` is after `Z` in ASCII order).

Reply:
It'd've also been prudent to add the names of those already listed in the same section (or at least check if they'd want to be).

A

Reply:
Well, if you want to be added you can also move the entry up as well!

Reply:
I'll open a PR (but I'm meant to be on holiday, so it might be a few hours!)

Reply:
Opened https://github.com/python/cpython/pull/138628